Formula/civl.rb

```diff
@@ -0,0 +1,23 @@
+class Civl < Formula
+  desc "The Concurrency Intermediate Verification Language"
+  homepage "http://vsl.cis.udel.edu/civl/"
+  url "http://vsl.cis.udel.edu/lib/sw/civl/1.7/r3157/release/CIVL-1.7_3157.tgz"
+  version "1.7-3157"
+  sha256 "33fa5b4877fbf494b0a4b426c15d6a165fa3db48a9f97cbf391818d965d2dfb9"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+  depends_on "z3"
+  depends_on "cvc4"
+
+  def install
+    libexec.install "lib/civl-1.7_3157.jar"
```

Comment:
@UniqMartin based on Civl documentation, I can run one of the example as a test. This is from the civl documentation:
_" To test your installation, copy the le examples/concurrency/locksBad.cvl to your work-
ing directory. Look at the program: it is a simple 2-process program with two shared variables
used as locks. The two processes try to obtain the locks in opposite order, which can lead
to a deadlock if both processes obtain their rst lock before either obtains the second. Type
\civl verify locksBad.cvl". You should see some output culminating in a message
The program MAY NOT be correct. See CIVLREP/locksBad_log.txt
"_

So do I need to write the command line since we already copy the files? Like this:

system "verify #{bin}/examples/concurrency/locksBad.cvl"